RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/utils/trie/Trie.java

```diff
@@ -9,7 +9,7 @@
 
     public Trie(List<String> words) {
```

Comment:
why it is called Trie?

Reply:
my question also

Reply:
![image](https://user-images.githubusercontent.com/18403426/125062450-115ee200-e0b7-11eb-8812-479be87ddac6.png)
I suppose that is why Andrew named it this way